### ISSUE TITLE
I notice this

### ISSUE THREAD

**Issue by PsyBooty** [jshackles/idle_master#297]
I noticed that by pausing and returning the application he immediately wins the cards, so make a featcher to "pause" and unpause the sistem from time to time ?

**Comment by jonas-med-ett-s** [comment]
That is correct. This feature is available in my fork: [Idle Master Extended](https://github.com/JonasNilson/idle_master_extended/releases) - File > Settings > Fast mode
